contracts/Administratable.sol

```diff
@@ -32,7 +33,7 @@ contract Administratable {
    */
   modifier onlyAdminOrRole(address adminContractAddress, IEndaomentAdmin.Role role) {
     EndaomentAdmin endaomentAdmin = EndaomentAdmin(adminContractAddress);
```

Comment:
Can we validate neither sender or adminContractAddress are 0? Also will require testing.

Reply:
Same thing for onlyAdmin modifier.

Reply:
For C01 we'll be refactoring this contract a bit and adding a constructor to it, so while we *can* do this I don't think there is much value in doing so

Reply:
Can you add it? I know its pedantic but I just want to make sure we are hitting the item as they specified. It should only add one line in each modifier.

Reply:
Added in 00712ff!